helix-core/src/main/java/org/apache/helix/task/TaskUtil.java

```diff
@@ -1036,6 +1037,46 @@ public static void purgeExpiredJobs(String workflow, WorkflowConfig workflowConf
     setNextJobPurgeTime(workflow, currentTime, purgeInterval, rebalanceScheduler, manager);
   }
 
+  /**
+   * The function that loops through the all existed workflow contexts and removes IdealState and
+   * Workflow Context if Workflow Config is missing.
```

Comment:
"workflow context of the workflow whose workflow config does not exist"

Reply:
Done.